src/controllers/users.controller.js

```diff
@@ -0,0 +1,70 @@
+const { errorHandler } = require('../helpers/errorHandler');
+const { usersService, initUserService } = require('../services/users.service');
+const { STATUS_CODES } = require('../variables/variables');
+const express = require('express');
+
+const usersController = () => {
+  initUserService();
+
+  const usersRoutes = express.Router();
+
+  usersRoutes.get('/', (req, res) => {
+    const users = usersService.getUsers();
+
+    res.status(STATUS_CODES.ok).send(users);
+  });
+
+  usersRoutes.post('/', (req, res) => {
+    try {
+      const user = req.body;
+
+      const newUser = usersService.createUser(user);
+
+      res.status(STATUS_CODES.created).send(newUser);
+    } catch (err) {
+      errorHandler(err, res);
```

Comment:
If you want to handle errors in one place use `next` callback and error handling middleware, so you don't need to pass res manually every time.
https://dev.to/qbentil/how-to-write-custom-error-handler-middleware-in-expressjs-using-javascript-29j1